docs/Style.thy

```diff
@@ -219,6 +219,102 @@ lemma test_lemma3:
          case_tac h; simp)
   done
 
+section \<open>Right vs left operator-wrapping\<close>
+
+text \<open>
+  When a term is too long, there is a general consensus to wrap it at operators. However, consensus
+  has never emerged on whether the operator should then end up at the end of the line (right
+  operator wrapping), or start of the next one (left operator wrapping).
+  Mathematicians have a tendency towards right-wrapping, programmers towards left-wrapping, they
+  each have advantages in specific contexts, thus both appear in the l4v proofs and are permitted
```

Comment:
Not sure it's maths vs programming, I've seen strong supporters of both styles in both communities.

Reply:
Happy with bland on that one :-)